inc/saiport.h

```diff
@@ -2517,6 +2517,19 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_POE_PORT_ID,
 
+    /**
+     * @brief Attach counter object list
+     *
+     * Counter object should be of type Selective.
+     * Fill (#SAI_COUNTER_ATTR_TYPE with #SAI_COUNTER_TYPE_SELECTIVE).
+     *
+     * @type sai_object_list_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_COUNTER
+     * @default empty
+     */
+    SAI_PORT_ATTR_SELECTIVE_COUNTER_LIST,
+
```

Comment:
We need to provide port object level control for enabling/disabling packet or bye count. 
With the current proposal 1 port can have multiple stats object (of type selective) where each stats object can be enabled independently for only packet, only byte or both. This though provides a fine grain control but do not allow coarse control where all the stats object for a given port can be consistently enabled for either packet or byte or both

Reply:
Thanks @JaiOCP for the review comments. As suggested, I have added new attribute to control counting of supported PKT/BYTE/Both stats at object level. 
Also I have updated the HLD with work flow. Pls review option 3.
PR is updated with other SAI modules which can use this functionality.